pkg/cmd/admin/router/router.go

```diff
@@ -642,6 +636,17 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		if errs := bulk.Create(list, namespace); len(errs) != 0 {
 			return errExit
 		}
+
+		if needsSCC {
+			fmt.Fprintf(out, "Router service account %q is not privileged to open ports on the host IP\n", cfg.ServiceAccount)
+		}
+
+		if sobjects, err := associateSecrets(kClient, secrets, namespace, cfg.ServiceAccount, objects); err != nil {
+			return err
+		} else {
+			objects = append(objects, sobjects...)
```

Comment:
This doesn't look right as it will add the objects twice wouldn't it? `sobjects` returned back from `associateSecrets` is all the secret objects added to the `objects` passed in.